app/views/notes/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <script src="/assets/notes.js" type="text/javascript"></script>
 
-<div class="col-md-9 note-show<% if @node.status == 4 || @node.status == 0 %> moderated<% end %>">
+<div class="col-md-12 note-show<% if @node.status == 4 || @node.status == 0 %> moderated<% end %>">
```

Comment:
Hm, ok! I see how this may help... however, do you think it could be potentially moving other page content around in a way that would cause trouble? I also wonder what might happen on other pages, such as wiki pages from `/app/views/wiki/show.html.erb` and `/app/views/question/show.html.erb` - other places this type of grid might be shown.

Reply:
er, i guess my note came out below 😄